static/script.js

```diff
@@ -69,3 +70,26 @@ function enableAutoRefresh () { // eslint-disable-line no-unused-vars
     }
   }
 }
+
+function unfoldAll () { // eslint-disable-line no-unused-vars
+  window.setInterval(unfoldButton, 100)
```

Comment:
I don't think it is a good idea to use one another interval.
This will produce race between refresh and unfold.
I would change the autorefresh function to respect unfoldButton instead of creating two separate intervals which fight with each other.